src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.springboot.datajpa.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.Data;
+
+@Data
```

Comment:
Lombok's `@Data` is very dangerous with entity classes
see here why
https://habr.com/ru/company/haulmont/blog/564682/